html/webappapis/scripting/events/event-handler-onmove-02.tentative.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<title>Window Object 'onmove'</title>
+<link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#handler-window-onmove">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<script>
+promise_test((t) => {
+  var expectedRect = {'x': 100, 'y': 200};
+  return new Promise(resolve => {
+      window.addEventListener("move", resolve);
```

Comment:
@javifernandez Please note that there is [no such event in the HTML spec](https://html.spec.whatwg.org/#windoweventhandlers).

As such the test needs to be rewritten or moved to get it passing.

Reply:
Oh, I missed that this is a tentative test. Please ignore.